services/core-api/app/api/securities/resources/bond.py

```diff
@@ -127,11 +128,16 @@ def put(self, bond_guid):
             raise BadRequest('permit_guid is required.')
         permit = Permit.find_by_permit_guid(permit_guid)
         if not permit:
-            raise BadRequest('No permit was found with the permit_guid provided.')
+            raise BadRequest(
+                'No permit was found with the permit_guid provided.')
         if permit.permit_guid == bond.permit.permit_guid:
-            raise BadRequest('This bond is already associated with this permit.')
-        if bond.permit.mine_guid not in [m.mine_guid for m in permit._all_mines]:
-            raise BadRequest('You can only transfer to a permit on the same mine.')
+            raise BadRequest(
+                'This bond is already associated with this permit.')
+        if bond.permit.mine_guid not in [
+                m.mine_guid for m in permit._all_mines
+        ]:
+            raise BadRequest(
+                'You can only transfer to a permit on the same mine.')
```

Comment:
I don't see any functional changes in this file?

Reply:
When I started the branch, I made changes to the bond model/resource and then removed all changes/pivoted the feature. This file is just formatting changes now which will hopefully be resolved once I update the formatting issue.